src/collective/collectionfilter/filteritems.py

```diff
@@ -94,7 +94,7 @@ def get_filter_items(
     groupby_criteria = getUtility(IGroupByCriteria).groupby
     idx = groupby_criteria[group_by]['index']
     current_idx_value = request_params.get(idx)
-    if not getattr(current_idx_value, '__iter__', False):
+    if not isinstance(current_idx_value, list):
```

Comment:
strings do have `__iter__` in python 3 ... doh

Reply:
the ``__iter__`` war probably meant to handle cases where ``current_idx_value`` was something iterable. I wonder if it wouldn't be better to do something like:
```
current_idx_value = current_idx_value or []  # handle None
if isinstance(current_idx_value, str):
    # do not expand a string to a list of chars
    current_idx_value = [current_idx_value]
else:
    try:
        current_idx_value = list(current_idx_value)
    except TypeError:
        # int and stuff.
        current_idx_value = [current_idx_value]
```

Reply:
yes ... this look more failsafe than simply checking for `list` ... I'll change that